src/pages/[platform]/build-a-backend/storage/download-files/index.mdx

```diff
@@ -1239,10 +1239,6 @@ Option | Type | Default | Description |
 
 ## Frequently Asked Questions
 
-- `downloadData` is cached; if you have recently modified a file you may not get the latest version right away. You can pass in `cacheControl: 'no-cache'` to get the latest version.
-- `downloadData` only returns the latest cached version of the file; there is [not yet an API to view prior versions](https://github.com/aws-amplify/amplify-js/issues/2131).
```

Comment:
I think these refers to S3 object versioning. We never supported these feature. But the wording is wrong. We should not mention 'cached'
```
- `downloadData` only returns the latest version of the file; there is [not yet an API to view prior versions](https://github.com/aws-amplify/amplify-js/issues/2131).
```